pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -74,6 +73,17 @@
         PLException,
     )
 
+    try:
+        from .lightning_gpu_qubit_ops import (
+            LightningGPUMPI_C128,
+            LightningGPUMPI_C64,
+            MPIManager,
+        )
+
+        MPI_Support = True
```

Comment:
Use upper case constants?
```suggestion
        MPI_SUPPORT = True
```

Reply:
updated in PR#112